lib/Command/UpdateGroup.php

```diff
@@ -0,0 +1,205 @@
+<?php
+/**
+ * @author Arthur Schiwon <blizzz@arthur-schiwon.de>
+ * @author Joas Schilling <coding@schilljs.com>
+ * @author Morris Jobke <hey@morrisjobke.de>
+ * @author Thomas Müller <thomas.mueller@tmit.eu>
+ *
+ * @copyright Copyright (c) 2016, ownCloud GmbH.
+ * @license AGPL-3.0
+ *
+ * This code is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License, version 3,
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>
+ *
+ */
+
+namespace OCA\User_LDAP\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use \OCA\User_LDAP\Helper;
+use \OCA\User_LDAP\LDAP;
+use \OCA\User_LDAP\Group_Proxy;
+use \OCA\User_LDAP\Mapping\GroupMapping;
+use \OCP\IDBConnection;
+use OCP\DB\QueryBuilder\IQueryBuilder;
+
+class UpdateGroup extends Command {
+
+	const ERROR_CODE_MISSING_CONF = 1;
+	const ERROR_CODE_MISSING_MAPPING = 2;
+
+	public function __construct(LDAP $ldap, Helper $helper, IDBConnection $connection) {
+		$this->connection = $connection;
+		$this->ldap = $ldap;
+		$this->helper = $helper;
+		parent::__construct();
+	}
+
+	protected function configure() {
+		$this
+			->setName('ldap:update-group')
+			->setDescription('update the specified group information')
+			->addArgument(
+					'groupID',
+					InputArgument::REQUIRED | InputArgument::IS_ARRAY,
+					'the group ID'
+				);
+	}
+
+	protected function execute(InputInterface $input, OutputInterface $output) {
+		$groupIDs = $input->getArgument('groupID');
+		// make sure we don't have duplicated groups in the parameters
+		$groupIDs = array_unique($groupIDs);
+
+		$helper = $this->helper;
+		$availableConfigs = $helper->getServerConfigurationPrefixes();
+
+		// show configuration information, useful to debug
+		$output->writeln('group membership attribute is critical for this command to work properly, please verify', OutputInterface::VERBOSITY_VERBOSE);
```

Comment:
I didn't want to keep bothering the user over and over when the attribute is properly set. However, adding the `--yes` option seems a better choice.